Source/WTF/wtf/text/WTFString.h

```diff
@@ -104,6 +104,8 @@ class String final {
     unsigned length() const { return m_impl ? m_impl->length() : 0; }
     const LChar* characters8() const { return m_impl ? m_impl->characters8() : nullptr; }
     const UChar* characters16() const { return m_impl ? m_impl->characters16() : nullptr; }
+    Span<const LChar> span8() const { return { characters8(), length() }; }
```

Comment:
Kinda feel like these should have the characters8/16 name long term and the raw pointers one can either go away or have a longer name. Not worth changing in this patch, but maybe worth a comment.

Reply:
I tried to be consistent with StringView which already had those:
```
    const LChar* characters8() const;
    const UChar* characters16() const;
    Span<const LChar> span8() const { return { characters8(), length() }; }
    Span<const UChar> span16() const { return { characters16(), length() }; }
```

Reply:
For future consideration, I like Sam’s idea even though I am the one who named the functions in `StringView`.